lib/flop_phoenix.ex

```diff
@@ -186,7 +186,8 @@ defmodule Flop.Phoenix do
     Default: `#{inspect(Pagination.default_opts()[:wrapper_attrs])}`.
   """
   @type pagination_option ::
-          {:current_link_attrs, keyword}
+          {:render_next_and_previous_links, boolean()}
+          | {:current_link_attrs, keyword}
```

Comment:
Little nitpick, the options here are in alphabetical order, can you move the new option to the right position?